puppet/modules/openvox_repo/manifests/apt.pp

```diff
@@ -0,0 +1,17 @@
+class openvox_repo::apt() {
+  include apt
+
+  $os_name = downcase($facts['os']['name'])
+  $release = "${os_name}${facts['os']['release']['major']}"
+
+  apt::source { "openvox":
+    comment  => "OpenVox ${openvox_repo::release} ${release} Repository",
+    location => 'https://apt.voxpupuli.org',
```

Comment:
I think the apt module will log a warning to your server log if that's a string and not an array (don't ask me why)